modules/ding_event/ding_event.info

```diff
@@ -19,6 +19,7 @@ dependencies[] = menu_position
 dependencies[] = number
 dependencies[] = og_ui
 dependencies[] = page_manager
+dependencies[] = pagepreview
```

Comment:
I do not think adding this module as a dependency here is needed.

The `ding_event` module does not have any configuration or code related to pagepreview so as I see it there are no actual dependencies between the two.

Since pagepreview is automatically enabled for all content types across the site I am fine with just enabling it in the install profile and leaving it at that.